ReleaseNotes.md

```diff
@@ -1,5 +1,55 @@
 # Release Notes
 
+## New in Release 2.5.0
+Compression-aware training:
+- New Features:
+  - Added Patcher class from OTX, which is used for patching the models that can not be wrapped properly.
+  - (PyTorch, Tensorflow) Added `strip` that convert compressed model to inference in backend format without NNCF specific operations (with int4 support).
+- Fixes:
+  - Fixed traced tensors to implement the YOLOv8 from Ultralytics.
+  - Fixed statistics computation for pruned layers.
+- Improvements:
+  - Extension of attributes (`transpose/permute/getitem`) for pruning node selector.
+  - NNCFNetwork was refactored from a wrapper-approach to a mixin-like approach.
+  - Added configuration for Accuracy Aware (pruning + quantization) for Resnet18 on CIFAR10.
+  - Added average pool 3d-like ops to pruning maskAdded Conv3d for overflow fix.
+  - Fixed JIT-traceable PyTorch models with internal patching.
+  - Added `__matmul__` magic functions to the list of patched ops (for SwinTransformer by Microsoft).
+  - Added `set_log_file()` method was added for NNCF logger.
+  - Added support for pruning of `torch.nn.functional.pad` operation.
+  - Added `torch.baddbmm` as an alias for the matmul metatype for quantization purposes.
+- Requirements:
+  - Updated ONNX version (1.13)
+- Docs:
+  - Multiple docs updates (fixes, improvements, etc.).
+
+Post-training compression:
+- Features:
+  - Added `"overflow_fix"` parameter support (improved accuracy for optimized model for affected devices).
+  - Added `"model_type"` parameter support that allows reach better accuracy for specific model architectures (e.g. transformers).
+  - Added `"inplace_statistics"` support (reduce memory footprint during optimization).
+  - Added `nncf.parameters.IgnoredScope` parameter support that allows exclude layers from optimization scope in different ways.
+  - (OpenVINO) Added weights compression after quantization (reduced .bin size of the optimized model).
+  - (OpenVINO) Added support of the Depthwise & GroupConvolution layers support for correct quantization.
+  - (OpenVINO) Added Quantization with accuracy control algorithm.
+  - (OpenVINO) Added YOLOv8 examples for [quantization](examples/post_training_quantization/openvino/yolov8) & [quantization with accuracy control](examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control).
+  - (OpenVINO, ONNX) Added BiasCorrection algorithm support.
+  - (OpenVINO) Added support for dynamic-shape models.
+  - (OpenVINO) Added GRU/LSTM quantization support.
+  - (OpenVINO) Added quantizer scales unification.
+  - (PyTorch) Added MinMaxAlgorithm support.
+- Fixes:
+  - Fixed several bugs which lead to performance drop (added a number of patterns, corrected quantizer parameters calculation, etc.).
+  - Checking correct ONNX opset version via the `nncf.quantize(...)`. For now, models with opset < 13 optimizes correctly in per-tensor quantization.
```

Comment:
(ONNX) `Fixed checking for correct ONNX opset version in `nncf.quantize(...)` calls with ONNX models - now models with opset < 13 will be optimized correctly during per-tensor quantization.